tests/MethodsParams.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MongoDB\Tests;
```

Comment:
This class is only relevant to the Unified Test Runner, so I don't think we need to have this in the top-level `MongoDB\Tests` namespace. That said, the class also has no methods on its own so I'd prefer to just move the array below to a `private static $operationArgs` array in the `MongoDB\Tests\UnifiedSpecTests\Util` class.